cypress.dev.json

```diff
@@ -1,7 +1 @@
-{
-  "baseUrl": "http://localhost:3000",
-  "env": {
-    "FOREM_OWNER_SECRET": "secret",
-    "ELASTICSEARCH_URL": "http://localhost:9200"
-  }
-}
+{}
```

Comment:
We still want this configuration file even though it's empty, because the one for CI, the default creates e2e test artifact folders in `/tmp` on the CI servers.